src/features/postApp/PostApp.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import {
+  fetchUsers,
+  selectUser,
+  usersSelector,
+} from '../userSelector/userSelectorSlice';
+import { selectPost } from '../postDetails/postDetailsSlice';
+import { fetchUserPosts, setPosts } from '../postsList/postsListSlice';
+import { UserSelector } from '../userSelector';
+import { PostsList } from '../postsList';
+import { PostDetails } from '../postDetails';
+
+export const PostApp: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const { selectedUser } = useAppSelector(usersSelector);
+  const selectedPost = useAppSelector(usersSelector);
+
+  const { userId } = useParams();
+
+  useEffect(() => {
+    dispatch(fetchUsers())
+      .unwrap()
+      .then(usersFromServer => {
+        if (userId) {
+          dispatch(selectUser(usersFromServer.find(user => user.id === +userId)
+          || null));
```

Comment:
maybe add `usersFromServer.find(user => user.id === +userId)
          || null))` to a variable?